SQL/MySQL.sublime-syntax

```diff
@@ -457,6 +457,7 @@ contexts:
         1: keyword.other.ddl.sql
         2: keyword.other.ddl.sql
       set:
+        - maybe-column-modifier
```

Comment:
Not all column modifiers are valid in that context.

`REFERENCES` seems to belong to `reference_definitions` which is supported after FOREIGN KEYS only.

We should maybe add related pattern directly to `alter-table-add-index-args`.

see: 

https://github.com/sublimehq/Packages/blob/77c99b4cbc4d2e65f65efd146184d3b2fd82d52d/SQL/tests/syntax/syntax_test_mysql.sql#L2413-L2415

Note: That's the section, related tests should be added to.

Reply:
We should keep closely to provided BNF representations, to not mix up things and cause other issues.

Reply:
Good point, I will try to bear that in mind 👍